content/docs/1.4.0/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,67 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass parameters to specify tags.
 
-For example:
-```
+When installing with Helm, the nodeSelector field of the default storage class `longhorn` can be enabled by editing `persistence.defaultNodeSelector` in [values.yaml](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/chart/values.yaml). Here are the steps as an example to deploy an Nginx pod with log data stored on specific node.
```

Comment:
NIT: This works for the Helm-installed Longhorn only. 

The most straightforward way is always editing the ConfigMap `longhorn-storageclass` in namespace `longhorn-system`.

Reply:
I have modified the way to add parameter of storage class with editing configmap named `longhorn-storageclass` directly. Please have a look to review the latest submit. Thank you.